### PR TITLE
Bookstore components

### PR DESCRIPTION
**Pull request by codecaiine**
- Create a new component branch
- Added React Redux 
- Structured the application files using a feature folder approach and ducks pattern 
- Added React Router
- Set Routes and Links for the app's navigation 
- Set up linters
- Fix linters errors

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
```
Comment (addod19) [inline src/components/BookList.js:2]: - [ ] It would be better if we create the Book component and render it in the BookList component

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @codecaiine, 

Good job so far! Project structure followed the duck's pattern :+1: Needed packages installed :smile: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookList.js (hunk 1 above)
Comment (codecaiine) [reply src/components/BookList.js:2]: Hi @addod19, 

Thank you for reviewing my work and your suggestions. I worked on it. 

Cheers!

**Review by e71az — APPROVED** [review]
## Status:  Approved 🟢

Hi, @, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Thank you @e71az for your approval .

**Pull request merged**